model_analyzer/analyzer.py

```diff
@@ -98,11 +104,22 @@ def profile(self, client, gpus):
 
         self._profile_models()
 
+        # The message is in interrupt_hanlder(), so we can just exit
+        if (self._state_manager.exiting()):
+            sys.exit(1)
+
+        # TODO: TMA-792: This won't be needed once the Results class is used in profile
+        self._analyze_models()
+
+        # TODO: TMA-790: Skip these steps if the CLI option indicates the user doesn't want summary reports
+        self._create_summary_tables(verbose)
+        self._create_summary_reports(mode)
+
         logger.info(self._get_profile_complete_string())
         logger.info("")
-        logger.info(self._get_analyze_command_help_string())
+        logger.info(self._get_report_command_help_string())
```

Comment:
have we verified this still works correctly?

Reply:
Yes, I've run profile and verified the output is correct